src/main/java/core/basesyntax/service/OperationStrategyImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operation.OperationHandler;
+
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy{
```

Comment:
The class name 'OperationStrategyImpl' should be followed by a space before the opening brace.